app/webpacker/js/turbo.js

```diff
@@ -0,0 +1,14 @@
+import "@hotwired/turbo";
+
+document.addEventListener("turbo:frame-missing", (event) => {
+  // don't replace frame contents
+  event.preventDefault();
+
+  // show error message instead
+  status = event.detail.response.status;
+  if(status == 401) {
+    alert(I18n.t("errors.unauthorized.message"));
+  } else {
+    alert(I18n.t("errors.general_error.message"));
+  }
+});
```

Comment:
Hmm, in Rails we arrange code like this in a folder called `initializers`. Maybe we could do the same with JS next time.